scavenger-frontend/src/components/snapshot/SnapshotTable.vue

```diff
@@ -158,6 +165,24 @@ export default {
           ElNotification.success({message: this.$t("message.snapshot.refresh-success")});
         })
     },
+    exportSnapshot(id) {
+      const fileName = `snapshot${id}.tsv`;
+      this.$http.get(`/customers/${this.customerId}/export/snapshot/${id}?fn=${fileName}`)
+        .then((response) => {
+          var csvFile = new Blob([response.data], {type: 'text/csv'});
+          var downloadLink = document.createElement("a");
```

Comment:
Plz use `let` or `const` instead of `var`.

Reply:
I fixed this in commit d969086.